resources/views/components/turnstile.blade.php

```diff
@@ -25,7 +25,7 @@ class="cf-turnstile"
 ></div>
 
 @if ($model)
-    <script>
+    <script @if(app('csp-nonce')) @cspNonce @endif>
```

Comment:
Does `app('csp-nonce')` return `null` if nothing is bound to the `csp-bounce` name? 

Can we also add a test for this please.